hotsos/core/plugins/storage/bcache.py

```diff
@@ -19,9 +19,11 @@ class BcacheConfig(ConfigBase):
 
     def get(self, key, section=None, expand_to_list=False):
         cfg = os.path.join(self.path, key)
-        if os.path.exists(cfg):
-            with open(cfg) as fd:
-                return fd.read().strip()
+        if not os.path.exists(cfg):
+            return None
+
+        with open(cfg) as fd:
```

Comment:
Following the same, wouldn't you want a final `return None` in case `open` fails?

Reply:
if open() fails it will throw an exception